platforms/chibios/flash_stm32.c

```diff
@@ -150,6 +164,39 @@ FLASH_Status FLASH_ErasePage(uint32_t Page_Address) {
     return status;
 }
 
+#if defined(STM32L4XX)
+/**
+ * @brief  Programs double words at a specified address.
+ * @param  Address: specifies the address to be programmed.
+ * @param  Data: specifies the data to be programmed.
+ * @retval FLASH Status: The returned value can be: FLASH_ERROR_PG,
+ *   FLASH_ERROR_WRP, FLASH_COMPLETE or FLASH_TIMEOUT.
+ */
+FLASH_Status FLASH_ProgramDoubleWord(uint32_t Address, uint64_t Data) {
```

Comment:
Seems needed only on dual bank flash, refer to "Data Cache might be corrupted during Flash Read While Write operation" in page 22 of [es0302-stm32l475xx-device-errata-stmicroelectronics.pdf](https://www.st.com/resource/en/errata_sheet/es0302-stm32l475xx-device-errata-stmicroelectronics.pdf)

I added data cache reset, since it may be used by a L4 mcu with dual bank.
We need do more test on it.

I have a few questions about the unit test:
1. I typed `make test:eeprom_stm32_large` and `make test:eeprom_stm32_tiny` and got the result in the QMK MSYS showing PASSED. I think it is not completed, we need to download the firmware to perform the test, how? I saw "eeprom_stm32_large.elf" in ./build/test folder. I tried to convert to hex with "arm-none-eabi-objcopy" , but it showed  " file format not recognized" error.
2. Current eeprom test doesn't not inlucde stm32l432, if i want to perform the test on stm32l432, I need to edited "rules.mk" in platforms/test  ,replace "eeprom_stm32.c" with "eeprom_stm32_l4.c" and add STM32L4XX to eeprom_stm32_DEFS, correct?